Source/ButtonContext.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cs505.group1.state;
+
+/**
+ * @author CS505 Fall17 Group1
```

Comment:
Make sure you add meaningful class level Javadoc for all of your classes.  This is particularly important for any code that you are expecting to be used by any developers other than the original team.

Reply:
Javadoc folder added and populated in the reorganize-folders branch.